frontend/src/test/java/org/abs_models/frontend/typesystem/LocationTypeTests.java

```diff
@@ -228,35 +227,22 @@ public void callReturn() {
     @Test
     public void callReturn2() {
         assertInferOk("module M.S1; export *; interface I { [Near] I m(); } class C implements I { [Near] I m() { return null; } } " +
-                      "module M.S2; import * from M.S1; { I i; i = new C(); i!m(); }", LocationType.FAR);
+            "module M.S2; import * from M.S1; { I i; i = new C(); i!m(); }", LocationType.FAR);
     }
-    
+
     @Test
     public void typeSyn() {
         assertInferOk("interface I {} type I2 = [Somewhere] I; { I2 i; i = null; }", LocationType.SOMEWHERE);
     }
-    
```

Comment:
Can you comment on why the test was removed?  (Inapplicable, fails with new implementation, ...)

Reply:
These two tests are based on the parameterized far types, which were removed. Without these far types, they fail.

Reply:
Ok, thanks.